release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3028,6 +3028,69 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Before this update, a hosted cluster rejected certificates with multiple Storage Area Network (SAN) entries due to conflicting Domain Name System (DNS) names. As a consequence, users encountered invalid configuration errors after deploying hosted clusters with the certificates. With this release, the cluster accepts certificates with multiple SAN entries. As a result, deployment errors are eliminated. (link:https://issues.redhat.com/browse/OCPBUGS-60485[OCPBUGS-60485])
+
+* Before this update, the Cluster Network Operator (CNO) missed setting the `release.openshift.io/version` annotation on a `DaemonSet` API object and in the `openshift-multus` namespace during an upgrade from {product-title} 4.18.6 to {product-title} 4.18.22. As a consequence, the upgrade failed due to missing annotations. With this release, the CNO sets the annotation for the `DaemonSet` object and for the deployment during an upgrade. As a result, upgrading a cluster completes without stopping due to missing annotations in the `openshift-multus` namespace. (link:https://issues.redhat.com/browse/OCPBUGS-60795[OCPBUGS-60795])
+
+* Before this update, improper machine deletion handling during cluster autoscaling caused the last node to retain the `ToBeDeletedByClusterAutoscaler` taint. As a consequence, resource allocation was affected during cluster scaling. With this release, the `ToBeDeletedByClusterAutoscaler` taint removal after scaling down a machine set is implemented. As a result, the last node does not retain the unwanted taint after scaling down a machine set. (link:https://issues.redhat.com/browse/OCPBUGS-60908[OCPBUGS-60908])
+
+* Before this update, disabling the {product-title} image registry left legacy pull secrets with persistent finalizers, which prevented their deletion. As a consequence, users were unable to delete `Dockercfg` secrets after disabling the image registry. With this release, legacy pull secrets do not block namespace deletion after the registry is removed. As a result, pull secret finalizers do not block namespace deletion when the registry is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-61002[OCPBUGS-61002])
+
+* Before this update, {ibm-cloud-title} was omitted from the list of platforms that supported single node installations in the validation code. As a consequence, users could not install a single node configuration on {ibm-cloud-title} because of a validation error. With this release, {ibm-cloud-title} support for single node installations is enabled. As a result, users can complete single node installations on {ibm-cloud-title}. (link:https://issues.redhat.com/browse/OCPBUGS-61178[OCPBUGS-61178])
```

Comment:
```suggestion
* Before this update, {ibm-cloud-title} was omitted from the list of platforms that supported single-node installations in the validation code. As a consequence, users could not install a single-node configuration on {ibm-cloud-title} because of a validation error. With this release, {ibm-cloud-title} support for single-node installations is enabled. As a result, users can complete single-node installations on {ibm-cloud-title}. (link:https://issues.redhat.com/browse/OCPBUGS-61178[OCPBUGS-61178])
```
debatable whether single-node as a compound modifier is acceptable, or if there is never a hyphen betwixt?